pkg/log/log.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package log
+
+type Level int32
+
+type Verboser interface {
```

Comment:
Hmmm true. Thanks for catching that. I'll add the linter at a later point because there are other packages missing comments that I don't want to touch in this PR.

Reply:
Fixed with https://github.com/kubernetes-sigs/minibroker/pull/72/commits/192bd1fe22304e71e8ae0f3c756562e83b5398e0.